data/schema_template.yaml

```diff
@@ -0,0 +1,24 @@
+source: the data provider: Eurostat, NOMIS, etc.
+api: the type of API to be used to fetch data: REST, download etc.
+api_documentation: the URL of a documentation page with descriptions of the field meaning (if available)
+endpoint_source: the URL of the endpoint where to actually run queries or the network requests for dumping the whole dataset
+enpoint_static: S3 URL of the data file
+description: a brief description of the data source and transformations that have been applied
+framework_id: indicator framework id (from data/framework.json) 
+is_experimental: True/False depending on whether this indicator is experimental
+year_range: [2006, 2018] #min and max year values
+nuts_level: 2
+schema:
+  nuts_id:
+    type: string
+  nuts_year:
+    type: int
+  year:
+    type: int
+    label: Year
+  value: #rename with the actual field name of the indicator value
```

Comment:
I think we agreed to use the actual string `value` as `id` would be there to specify the name of the key, so I'd remove the comment

Reply:
I have just changed this, but now having second thoughts. It might make sense for the app, but for a data scientist/ BEIS analyst working with this data later, it's quite a bit of hassle to be renaming columns with information from schemas.

It would make life a lot easier for them if the field name was also specified in the dataset, particularly if doing any joins/concatenations etc.

Maybe there's a way we can easily accommodate this use case and easy coding for the app?